cfgov/unprocessed/css/skip-nav.less

```diff
@@ -9,26 +9,29 @@
   tags:
     - cfgov-header
 */
+.skip-nav {
+  position: relative;
 
-#skip-nav {
+  &_link {
     position: absolute;
-    top: unit(-@grid_gutter-width / @base-font-size-px, em);
-    left: unit(@grid_gutter-width / @base-font-size-px, em);
+    top: auto;
+    left: -10000px;
     background: transparent;
     transition: transform 1s ease, background .5s linear;
     z-index: 11;
-}
 
-#skip-nav:focus {
-    .a-btn;
-    // Taken from .a-btn__super.
-    padding:
-      unit( 11px / 18px, em )
-      unit( 29px / 18px, em );
-    font-size: unit( 18px / @base-font-size-px, em );
+    &:focus {
+      .a-btn();
+      // Taken from .a-btn__super.
+      padding:
+        unit( 11px / 18px, em )
+        unit( 29px / 18px, em );
+      font-size: unit( 18px / @base-font-size-px, em );
 
-    position: absolute;
-    top: 0;
-    outline: 0;
-    transition: transform .1s ease, background .2s linear;
+      top: 10px;
+      left: 10px;
+      outline: 0;
+      transition: transform .1s ease, background .2s linear;
+    }
+  }
 }
```

Comment:
Please maintain the four-space indentation in this file.

Reply:
Fixed